flake.nix

```
@@ -175,6 +175,21 @@
                   "^RELEASES\\.md$"
                 ];
               };
+              clippy =
+                let
+                  allow = [
+                    "clippy::new-without-default"
+                    "clippy::match_like_matches_macro"
+                  ];
+                  allowOpts = builtins.concatStringsSep " "
+                    (builtins.map (lint: "-A \"${lint}\"") allow);
+                in
+                {
+                  enable = true;
+                  entry = pkgs.lib.mkForce ''
+                    ${rust}/bin/cargo-clippy clippy --frozen --offline -- -D warnings ${allowOpts}
```

It seems cargo doesn't like `--offline` in clippy, it cannot find `ansi_term` (presumably to recursively run clippy on it?). The same thing happens locally.
I'm not familiar with how the flake works, but it seems like passing `--offline` here is superfluous.
I suggest disabling the run on dependencies. Then it _shouldn't_ need to access to the network.

Nix sure is something.
```suggestion
                    ${rust}/bin/cargo-clippy clippy --frozen --no-deps -- -D warnings ${allowOpts}
```